flow/controllers/velocity_controllers.py

```diff
@@ -45,7 +44,6 @@ def __init__(self,
         self.d_1 = 1.5
         self.d_2 = 1.0
         self.d_3 = 0.5
-        self.danger_edges = danger_edges if danger_edges else {}
```

Comment:
i removed danger edges here, because it was not documented, and never used anywhere